profile/README.md

```diff
@@ -1,7 +1,9 @@
 ## We are AlmaLinux
 
-AlmaLinux OS is replacing CentOS as the downstream rebuild of RedHat Enterprise Linux. 
+AlmaLinux is a free linux OS for the community, by the community. 
 
-A healthy community is required for any open source project, and we are working on expanding ours. We will be sharing our infrastructure, event, and other outreach and improvement plans on the [AlmaLinux OS blog](https://almalinux.org/blog) soon!
+An Open Source, community owned and governed, forever-free enterprise Linux distribution, focused on long-term stability, providing a robust production-grade platform. AlmaLinux OS is ABI compatible with RHEL®.
 
-If you want to contribute to AlmaLinux, there are tons of ways to do so! We need help with everything from [website updates](https://github.com/AlmaLinux/almalinux.org) to marketing and event staffing, to infrastructure automation (in Infra team even has a list of things they're [working on in the wiki](https://wiki.almalinux.org/sigs/Infrastructure.html#help-wanted)).
\ No newline at end of file
+AlmaLinux is an enterprise linux distribution build with the intention of providing users of historic CentOS Linux with a replacement that meets their needs by being stable, secure, and ABI compatible with RedHat Enterprise Linux®. There are exciting developments all the time, and you can stay up to date by joining our [chat](https://chat.almalinux.org), or watching the [AlmaLinux OS blog](https://almalinux.org/blog) for updates!
+
+If you want to contribute to AlmaLinux, there are tons of ways to do so! We need help with everything from [website updates](https://github.com/AlmaLinux/almalinux.org) and marketing and event staffing, to infrastructure automation (the Infra team even has a list of things they're [working on in the wiki](https://wiki.almalinux.org/sigs/Infrastructure.html#help-wanted)), and help improving the AlmaLinux Build System (ALBS), but the best place to start is by joining our [chat](https://chat.almalinux.org) on mattermost. 
```

Comment:
Looks good!

I would consider moving the https://wiki.almalinux.org/sigs/Infrastructure.html#help-wanted link to the Infrastructure OpenProject as it better reflect what's going on: https://openproject.almalinux.org/projects/infrastructure/work_packages

100% your discretion though!

Reply:
Love that idea!